server/models/Profile.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose");
+const timeSlot = new mongoose.Schema({ start: Date, end: Date })
+
+const profileSchema = new mongoose.Schema({
+    firstname: { type: String },
```

Comment:
Would like some validation and sanitization on these fields. `required`, `trim`, etc.

Reply:
Change throughout the schema please 👍🏻

Reply:
You right, i will put sanitazion and requirement here